tests/unit/data-access/MwApiItemSearcher.test.ts

```diff
@@ -0,0 +1,95 @@
+import { MwApi } from '@/@types/mediawiki';
+import { WbSearchEntitiesResponse } from '@/data-access/apiBasedItemSearcher';
+import { SearchedItemOption } from '@/data-access/ItemSearcher';
+import MwApiItemSearcher from '@/data-access/MwApiItemSearcher';
+import unusedApi from '../../mocks/unusedApi';
+
+describe( 'MwApiItemSearcher', () => {
+
+	it( 'gets and transforms search results', async () => {
+		const api: MwApi = {
+			...unusedApi,
+			get: jest.fn().mockResolvedValue( {
```

Comment:
I'm somewhat torn between adding that code now and removing it again in a few days. But it is already written, so we might as well add it.

Reply:
I don’t follow, why would we remove it again?

Reply:
removing in the sense of basically completely rewriting, because we're targeting the new version of the API response that has the `display` key for us to use.